setup.py

```diff
@@ -11,29 +11,31 @@
 
 requirements = [
     'Click>=6.0',
-    # TODO: put package requirements here
+    'bigchaindb-common>=0.0.3',
+    'graphviz'
+]
```

Comment:
I mean I put `.` such that `pip install -r requirements_dev.txt` effectively also performs `pip install .`.